src/SpeedJumpIrm.sol

```diff
@@ -167,7 +167,14 @@ contract AdaptativeCurveIrm is IIrm {
     function _curve(int256 _rateAtTarget, int256 err) private view returns (int256) {
         // Non negative because 1 - 1/C >= 0, C - 1 >= 0.
         int256 coeff = err < 0 ? WAD - WAD.wDivDown(CURVE_STEEPNESS) : CURVE_STEEPNESS - WAD;
-        // Non negative because if err < 0, coeff <= 1.
+        // Non negative if _rateAtTarget >= 0 because if err < 0, coeff <= 1.
         return (coeff.wMulDown(err) + WAD).wMulDown(int256(_rateAtTarget));
     }
+
+    /// @dev Returns the new rate at target, for a given `startRateAtTarget` and a given `linearAdaptation`.
+    /// The formula is the following: newRAT = max(min(startRAT * exp(linearAdaptation), MAX_RAT), MIN_RAT).
```

Comment:
```suggestion
    /// Returns max(min(startRateAtTarget * exp(linearAdaptation), maxRateAtTarget), minRateAtTarget).
```
[minor] because if find using "RAT" a bit ugly :D